src/controllers/questionController.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable require-jsdoc */
+import model from '../database/models';
+
+const { Question } = model;
+
+export default class questionController {
+  // get all questions
+  static async getAllQuestions(req, res) {
+    try {
+      const allQuestions = await Question.findAll({});
+
+      return res.status(200).json({ message: res.__('All questions found successfully!'), allQuestions });
+    } catch (error) {
+      return res.status(500).json({ error: error.message });
+    }
+  }
+
+  // get one Question
+  static async getOneQuestion(req, res) {
+    try {
+      const { id } = req.params,
+        existingQuestion = await Question.findOne({ where: { id } });
+
+      return res.status(200).json({ message: res.__('Question found successfully!'), existingQuestion });
+    } catch (error) {
+      return res.status(500).json({ error: error.message });
+    }
+  }
```

Comment:
What if no question with the specified id found? It automatically end up in catch block, you should first check the existence of question with that id.

Reply:
We also need endpoint of deleting questions, because the case may happen that questions need to be deleted.